src/main/conversions/c2_16_0_2017091901.clj

```diff
@@ -0,0 +1,20 @@
+(ns facepalm.c2-16-0-2017091901
+  (:use [korma.core :exclude [update]]
+        [kameleon.uuids]))
+
+(def ^:private version
+  "The destination database version"
```

Comment:
Hmm, maybe this should say something like. "Arrrrrggghhhh!!! This be the destination database version ye scurvy dog!" 😛